### ISSUE TITLE
False positive for SpaceInsideHashLiteralBraces

### ISSUE THREAD

**Issue by dominikh** [rubocop/rubocop#271]
SpaceInsideHashLiteralBraces with EnforcedStyleIsWithSpaces == false will complain for the following snippet:

``` ruby
a = {
  :a => 1,
  :b => 2,
}
```

Multi-line hash literals should be handled extra.
